conf/messages

```diff
@@ -163,11 +163,37 @@ departure.details.error.overdue = Date and time of departure cannot be more than
 departure.details.error.future = Date and time of departure cannot be in the future
 
 location.title = Location
-location.question = What is the code for the location of the goods when they are declared?
-location.hint = This is usually the port they are leaving from and is officially known as the Goods Location Code. For example, if they were at Felixstowe, it would be GBAUFXTFXTFXT.
+location.question = What is the goods location code?
+location.hint = <p>This is a code for the location where goods are presented to customs. If necessary the goods location code must be updated as part of the arrival process, when the goods arrive at that location.</p><p>Most goods location codes contain just letters. For example, for goods leaving from Felixstowe, it would be GBAUFXTFXTFXT.</p>
```

Comment:
I don't think it is a good idea to put html tags into messages. This should be done rather as partials or components. The visibility of styling/structure of content is worse and you'd need to look into multiple places to figure that out.

Reply:
Indeed. Yyou can easily have two `@paragraphBody`s inside braces and assigned to a wrapping var.

Reply:
Ok, I've genericides the exportsInputText component to allow `Content` to be passed as the 'hint' value rather than a messages key. It is a bit frustrating having to add a whole other message key-value pair just to put a space between two lines of content! :wink: